packages/uniforms-antd/src/TextField.tsx

```diff
@@ -14,6 +14,7 @@ function Text(props: TextFieldProps) {
   return wrapField(
     props,
     <Input
+      autoComplete={props.autoComplete}
```

Comment:
Is this one actually needed? It should not as `{...filterDOMProps(props)}` below does not remove `autoComplete`. Same goes for `uniforms-material` theme.

Reply:
I wanted to use it with _-unstyled_ but it was assigned to `<div>` instead of `<input>`

Reply:
Yeah, I see that both `uniforms-semantic` and `uniforms-unstyled` required this change, hence I'm asking about `uniforms-antd` and `uniforms-material`. Also, what about `uniforms-bootstrap3` and `uniforms-bootstrap4`? Could you add tests there as well?

Reply:
I'm not sure about _-antd_ and _-material_, but I've expected same behavior from them as _-unstyled_. I've missed the bootstrap versions, so I'll look at them also.

Reply:
So, in new commits, tests are added and if it is not needed to explicitly set the value it is not there.